test/spec/widget.analog-scale.spec.js

```diff
@@ -10,3 +10,68 @@ const FORM2 = FORM1.replace( 'step="1"', 'step="0.1"' );
 
 runAllCommonWidgetTests( AnalogScaleWidget, FORM1, '2' );
 runAllCommonWidgetTests( AnalogScaleWidget, FORM2, '2.1' );
+
+const FORM_SHOW_SCALE =
+    `<label class="question non-select or-appearance-analog-scale or-appearance-vertical or-appearance-show-scale">
+        <span lang="" class="question-label active">Analog Scale</span>
+        <input type="number" name="/widgets/analog_scale/widget1" data-type-xml="int">
+    </label>`;
+
+testStaticProperties( AnalogScaleWidget );
```

Comment:
`testStaticProperties` is already called (twice) with runAllCommonWidgetTests.

Reply:
Got it, will remove that.